template/templates/common/item_meta.html

```diff
@@ -4,6 +4,11 @@
         <li class="item-meta-info-title">
             <a href="{{ route "feedEntries" "feedID" .entry.Feed.ID }}" title="{{ .entry.Feed.SiteURL }}" data-feed-link="true">{{ truncate .entry.Feed.Title 35 }}</a>
         </li>
+        {{ if .entry.Author }}
+        <li class="item-meta-info-author">
+            <em title="{{ .entry.Author }}">{{ truncate .entry.Author 35 }}</em>
```

Comment:
What is the semantic meaning for using `<em>`? If the answer is: to visually style the element in italics, then this should prefer CSS over using the `<em>` element.

Reply:
To be honest, I had no specific reason to choose `<em>` over something less semantic :grimacing: 

If you think it is inappropriate I am happy to (have you) change it to e.g. `<span>` or remove the extra inline element completely, if that would not break stuff.

Reply:
```css
.item-meta-info-author {
	font-style: italic;
}
```

is all that would be needed.